tests/Exception/Renderer/InlineDebugRendererTest.php

```diff
@@ -33,7 +33,7 @@ public function testRenderWithoutDebug(): void
         $debug = false;
         $exception = $this->createMock('\Exception');
         $block = $this->createMock('Sonata\BlockBundle\Model\BlockInterface');
-        $templating = $this->createMock('Symfony\Bundle\FrameworkBundle\Templating\EngineInterface');
+        $templating = $this->createMock('Twig\Environment');
```

Comment:
Can you use ::class notation? Only on the lines you changed. No need to fix the whole file

Reply:
Done. Also i renamed all `$templating` to `$twig` in test classes